java/clients/src/test/java/sleeper/clients/status/report/compaction/job/CompactionJobStatusReporterTestBase.java

```diff
@@ -70,49 +70,43 @@ protected static List<CompactionJobStatus> mixedJobStatuses() {
         CompactionJob job2 = dataHelper.singleFileCompaction(partition("B"));
         Instant creationTime2 = Instant.parse("2022-09-18T13:33:12.001Z");
         Instant startedTime2 = Instant.parse("2022-09-18T13:34:12.001Z");
-        CompactionJob job3 = dataHelper.singleFileCompaction(partition("C"));
-        Instant creationTime3 = Instant.parse("2022-09-19T13:33:12.001Z");
-        Instant startedTime3 = Instant.parse("2022-09-19T13:34:12.001Z");
```

Comment:
This is a really minor nitpick, but the partition IDs and times were previously consecutive, i.e. A, B, C D... and 18th Sept, 19th Sept, 20th Sept...

It's missing C and 19th September from the sequence now.